cli/cmd/upgrade_test.go

```diff
@@ -186,6 +189,11 @@ func TestUpgradeOverwriteIssuer(t *testing.T) {
 			if id == "ConfigMap/linkerd-config" {
 				continue
 			}
+
+			if id == "Deployment/linkerd-identity" {
+				// skip checking identity as `identity-trust-anchor-pem` is overridden, which is passed as a CLI flag
```

Comment:
if we wanted to take this a step further, we could verify that that the path is `{spec, containers, *, spec, args, *}` or whatever and that the `diff.b` is `issuerCerts.caCrt`.

Reply:
Great suggestion! Done :+1: